backend/graphql/types/lessonType.ts

```diff
@@ -28,6 +28,8 @@ const lessonType = gql`
 
   extend type Mutation {
     createLesson(lesson: LessonRequestDTO!): LessonResponseDTO!
+    updateLesson(id: ID!, lesson: LessonRequestDTO!): LessonResponseDTO!
```

Comment:
Regardless of if create and update use the same DTO or different ones, I think we need a more clear name than `LessonRequestDTO`, it comes off as a DTO to get a lesson. Something like `LessonCreateRequestDTO` or `LessonCreateUpdateRequestDTO` (if we choose to use the same DTO` would be more appropriate imo.